docs/user-guides/acme-http-validation.md

```diff
@@ -0,0 +1,96 @@
+# Issuing an ACME certificate using HTTP validation
+
+`cert-manager` can be used to obtain certificates from a CA using the [ACME][1] protocol. The ACME protocol supports various challenges which are used to prove ownership of a domain so that a valid certificate can be issued for that domain. Perhaps the most simple is the HTTP challenge which requires that a file must be made available at the domain that is to be validated. It is assumed that you control the domain if you are able to publish a given file under a given path.
```

Comment:
Can we remove "perhaps the most simple"? We don't want to pass a judgement as to which is best - it's dependent on each user. Many find DNS01 simpler too 😄